content/conditions/hernia/surgery-guide/manifest.json

```diff
@@ -6,141 +6,77 @@
   "content": {
     "header": [
       {
-        "type": "article_nav"
+        "type": "text",
+        "props": {
+          "variant": "markdown",
+          "value": "!file=local-header.md"
+        }
       }
     ],
     "main": [
       {
-        "type": "article_list",
+        "type": "text",
+        "props": {
+          "variant": "markdown",
+          "value": "!file=section-1.md"
+        }
+      },
+      {
+        "type": "text",
+        "props": {
+          "variant": "markdown",
+          "value": "!file=section-2.md"
```

Comment:
Do we still need them as separate text components? This could be just a single Markdown block.